### PR TITLE
Remove hardcoded rule-api version from rspec.ps1

### PR DESCRIPTION
**Pull request by andrei-epure-sonarsource**

### CODE REVIEW

File: scripts/rspec/rspec.ps1 [hunk 1]
```diff
@@ -321,7 +324,7 @@ function ReplaceTokens($text) {
 ### SCRIPT START ###
 
 $sonarpediaFolder = $sonarpediaMap.Get_Item($language)
-Write-Host "Will change directory to $sonarpediaFolder to run rule-api"
+Write-Host "Will change directory to $sonarpediaFolder to run rule-api (from $rule_api_jar )"
```
Comment (andrei-epure-sonarsource) [inline scripts/rspec/rspec.ps1:327]: I left a space after the path to avoid copy-paste problems

File: scripts/rspec/rspec.ps1 [hunk 2]
```diff
@@ -36,14 +36,17 @@ $ErrorActionPreference = "Stop"
 $RuleTemplateFolder = "${PSScriptRoot}\\rspec-templates"
 
 # Update the following variable when a new version of rule-api has to be used.
-$rule_api_version = "1.24.2.1313"
-$rule_api_error = "Download Rule-api from " + `
-    "'https://repox.jfrog.io/repox/sonarsource-private-releases/com/sonarsource/rule-api/rule-api/${rule_api_version}' " +`
+$rule_api_error = "Could not find the Rule API Jar locally. Please download the latest rule-api from " + `
```
Comment (andrei-epure-sonarsource) [inline scripts/rspec/rspec.ps1:39]: ideally we'd download it, but we need to authenticate and I'm not sure it's worth the effort

File: scripts/rspec/rspec.ps1 [hunk 3]
```diff
@@ -36,14 +36,17 @@ $ErrorActionPreference = "Stop"
 $RuleTemplateFolder = "${PSScriptRoot}\\rspec-templates"
 
 # Update the following variable when a new version of rule-api has to be used.
```
Comment (csaba-sagi-sonarsource) [inline scripts/rspec/rspec.ps1:38]: This comment is not needed anymore.

**Review by csaba-sagi-sonarsource — APPROVED** [review]
LGTM. Please remove the comment that was meant for the variable that was removed.

**Pull request merged**